app/grandchallenge/core/templates/base.html

```diff
@@ -101,6 +104,9 @@
 {% block script %}
     {% include "grandchallenge/partials/script.html" %}
     {% include "timezones/partials/detect_timezone.html" %}
+    {% if challenge %}
+        <script src="{% static 'js/challenges/dropdown.js' %}" type="module"></script>
```

Comment:
This script line can move to app/grandchallenge/challenges/templates/challenges/challenge_topbar.html, as long as it's deferred with `<script src="{% static 'js/challenges/dropdown.js' %}" type="module" defer></script>`